00_hello_world/juliaogris/main_test.go

```diff
@@ -4,21 +4,18 @@ import (
 	"bytes"
 	"strconv"
 	"testing"
-
-	"github.com/stretchr/testify/require"
 )
 
 func TestMainOutput(t *testing.T) {
-	// Given
-	r := require.New(t)
 	var buf bytes.Buffer
 	out = &buf
 
-	// When
 	main()
 
-	// Then
 	expected := strconv.Quote("Hallo du schöne Welt!\n")
 	actual := strconv.Quote(buf.String())
```

Comment:
I've read a few times `got` and `want` instead of `actual` and `expected`- up to you if want to change it.
See:

 - https://github.com/golang/go/wiki/TableDrivenTests#example-of-a-table-driven-test
- https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

Reply:
I really don't think there's much of a difference, less characters I suppose.